formulaire/src/main/java/fr/openent/formulaire/export/FormQuestionsExportPDF.java

```diff
@@ -174,6 +174,12 @@ public void launch() {
                                     if (Objects.equals(child.getInteger(MATRIX_ID), question.getInteger(ID))) {
                                         if (question.containsKey(CHILDREN)) {
                                             question.getJsonArray(CHILDREN).add(child);
+                                            if (child.getInteger(QUESTION_TYPE).equals(QuestionTypes.SINGLEANSWERRADIO.getCode())) {
```

Comment:
```suggestion
                                            if (QuestionTypes.SINGLEANSWERRADIO.getCode().equals(child.getInteger(QUESTION_TYPE))) {
```
Comme ça, ça te garanti d'éviter un null pointer ;)

Reply:
Bien vu merci